app/src/main/java/com/infomaniak/mail/data/LocalSettings.kt

```diff
@@ -66,6 +66,7 @@ class LocalSettings private constructor(context: Context) : SharedValues {
     var isMatomoTrackingEnabled by sharedValue("isMatomoTrackingEnabledKey", true)
     var autoAdvanceMode by sharedValue("autoAdvanceModeKey", AutoAdvanceMode.THREADS_LIST)
     var autoAdvanceNaturalThread by sharedValue("autoAdvanceNaturalThreadKey", AutoAdvanceMode.FOLLOWING_THREAD)
+    var canShowWebViewOutdated by sharedValue("canShowWebViewOutdated", true)
```

Comment:
Would "hasWebViewOutdatedBeenDismissed" or something in this direction be more appropriated?

Reply:
Above, we also have `canShow...` that's why I prefered to keep the same logic.

Reply:
Maybe `show...` then

Reply:
Boolean should start with can, should, has, etc…

Reply:
I've unresolved this conversation.
A boolean should start with is, can, should, has, etc…